### ISSUE TITLE
Nodes reporting uptime without relay/public key set

### ISSUE THREAD

**Issue by scottyeager** [threefoldtech/zos#2027]
Today I noticed that there are a few nodes that are creating uptime reports while not having a relay and public key set on chain for the node's twin.

These are the node ids:

```
3830
3814
2988
```

Node 3830 is logging quite a lot of errors in this general form:

```
2023-08-23 20:44:24 | [+] noded: 2023-08-23T18:44:24Z error failed to decode events from tfchain error="unable to find event with EventID [216 147] in metadata for event #1: module index 216 out of range"
```

The other two nodes don't seem to be submitting any logs to Loki (I ran the query up to 30 days history range). For these nodes there's probably not much to do, aside from block them from minting.

Maybe for the one node logging errors, there's something to look into.

**Comment by muhamadazmy** [comment]
I checked the logs and those nodes looks very weird, some how they only report logs from one module  that looks awfully out of date!

One possible explanation is that those nodes were running a very old image, then failed to update at some point and got stuck with only some of the services running with no way to re correct itself. Most of those issues were fixed in later releases but seems those nodes were stuck in a bad state.

If possible i would ask the farmer to reboot them so they get latest available version.
Also do they mint ? i remember there was some rules in minting to stop nodes who don't have relays set from minting.

The other nodes that are not reporting logs can also be affected similarly to how explained above but with even (vector) down! another explanation that is a little bit far fetched is that those node with no logs are fake nodes which only report uptimes
